### PR TITLE
Contribution: Importing chemical data to Excel

### PR DESCRIPTION
**Pull request by stuchalk**
This contribution focuses on demonstrating how chemical data can be imported into Excel and provides some hints and tips of how format data and import it without errors.  Note, four of the images are replacements for ones were already uploaded.

### CODE REVIEW

**Review by kroenlein — COMMENTED** [review]
I got a few sections in, and hit a mental road bump around plain-text v. binary and encodings.  I think those definitions need to be front loaded.  Can you insert 1-2 sentences explaining those terms and contexts before jumping into the bulleted file formats?

As a side note, if you format a .md file with each sentence on a new line (or multiple lines), it makes it a lot cleaner to make suggestions, since then the diffs render with full context of the proposed changes.

File: book/school/cs_spreadsheet_data_import.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
+# Importing Chemical Data into a Spreadsheet
+
+```{dropdown} Culinary School Topic
+- Author: [Stuart Chalk](https://orcid.org/0000-0001-6513-9140)
+- Reviewer: 
+- Topics: Data Importing, Spreadsheets, Chemical Data
+- Format: Markdown file
+- Skills: You should be familiar with
+    - [Computer files and extensions](https://www.howtogeek.com/356448/what-is-a-file-extension/)
```
[the remark this answers is not in the document]
Comment (stuchalk) [reply book/school/cs_spreadsheet_data_import.md:9]: OK by me...

File: book/school/cs_spreadsheet_data_import.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    - How to import data into Microsoft Excel and Google Sheets
+    - Understand the difference between .csv, .tsv and .txt files
+    - Identify why data may be corrupted upon import a file
+- Citation: 'Importing Chemical Data into Spreadsheet', The IUPAC FAIR Chemistry Cookbook, 
+  https://iupac.github.io/WFChemCookbook/school/cs_spreadsheet_data_import.html
+- Reuse: This notebook is made available under a [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/) license.
+```
+
+## Summary
+This tutorial focuses on importing data into spreadsheets.  This is a very common activity for chemists that results from
```
[the remark this answers is not in the document]
Comment (stuchalk) [reply book/school/cs_spreadsheet_data_import.md:22]: +1

**Comment by stuchalk** [comment]
I have added paragraphs about text v's binary files in commit 29c772d .

File: book/images/spreadsheet_fig1.jpg
Comment (Smund27) [inline book/images/spreadsheet_fig1.jpg:1]: What was the change here? Happy to accept it but want to double check the correct change was implemented

File: book/images/spreadsheet_fig2.jpg
Comment (Smund27) [inline book/images/spreadsheet_fig2.jpg:1]: Was there meant to be a change here?

**Review by Smund27 — COMMENTED** [review]
This looks very good, happy to approve once have clarification over two changes that don't seem to have changed anything significant.

**Comment by stuchalk** [comment]
Looking at the PR I need to pull this and start again.  There are lot of other files that should not be in this PR and it is clear I did not know what I was doing when I submitted this.

**Pull request closed without merging**